### DISCUSSION TITLE
Boolean labels do not show in collection filters

### DISCUSSION THREAD

**Discussion by MortenHofft** [directus/directus#6465] [Feature Requests]
Fresh install. I'm on 9.0.0-rc.80. Using a local postgres database. And I'm a first time user with a total of 10 minutes experience.

I can nicely change my label for boolean values. I changed mine to `Is invasive`. But when filtering the default `Enabled` is used instead. I imagine this is a mistake.

<img width="214" alt="Screenshot 2021-06-23 at 14 34 05" src="https://user-images.githubusercontent.com/6759267/123097181-29383400-d430-11eb-9784-b6d1f35f233f.png">
<img width="1182" alt="Screenshot 2021-06-23 at 14 23 39" src="https://user-images.githubusercontent.com/6759267/123097209-305f4200-d430-11eb-9e8d-13b58f7d4514.png">

btw: Looks like a wonderful product

**Comment by benhaynes** [comment]
Hey, and welcome to the Directus community! :)

This is actually intended behavior... for the moment. The filters use the field **type** information, but not the interface info. However this is something that we're actually working on today, so perhaps this can be updated too. Our current focus is pulling in the choices for dropdown, but we could look into other options as well. Thoughts @rijkvanzanten ?

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
